readthedocs/search/indexes.py

```diff
@@ -143,7 +142,7 @@ def bulk_index(self, data, index=None, chunk_size=500, parent=None,
             docs.append(doc)
 
         # TODO: This doesn't work with the new ES setup.
-        bulk_index(self.es, docs, chunk_size=chunk_size)
+        # bulk_index(self.es, docs, chunk_size=chunk_size)
 
     def index_document(self, data, index=None, parent=None, routing=None):
         doc = self.extract_document(data)
```

Comment:
Guessing this entire file and other related code should be deleted?

Reply:
Yes. it need to be deleted. I will delete once I implement the file searching functionality!